### ISSUE TITLE
ValueError: Error raised by bedrock service: An error occurred (ValidationException) when calling the InvokeModel operation: The requested operation is not recognized by the service.

### ISSUE THREAD

**Issue by NingLu** [aws-samples/generative-ai-amazon-bedrock-langchain-agent-example#4]
After deployment, i type in Demo User and 1234, and can see 3 buttons in the Lex bot. Then typing "Good morning", and see this error

It is raised in this code snippet

```
    llm = Bedrock(
        model_id="anthropic.claude-instant-v1"
    )  
    llm.model_kwargs = {'max_tokens_to_sample': 200}
    lex_agent = FSIAgent(llm, chat.memory)

    try:
        message = lex_agent.run(input=prompt)
```

<img width="1550" alt="Screenshot 2023-10-18 at 21 46 58" src="https://github.com/aws-samples/generative-ai-amazon-bedrock-langchain-agent-example/assets/7640872/7d186f1c-526c-43b5-ac04-36c703f07b9a">

**Comment by awsartisan** [comment]
I'm getting the same error. I had hoped that pip install -U langchain might fix this, but it didn't.

**Comment by kyleblocksom** [comment]
I was not able to replicate the error in my environment, however, I reintroduced _FORMAT_INSTRUCTIONS_ into the [fsi_agent.py](https://github.com/aws-samples/generative-ai-amazon-bedrock-langchain-agent-example/blob/main/agent/lambda/agent-handler/fsi_agent.py) class. Please re-test and let me know your findings.

**Comment by dgghosalaws** [comment]
The issue is still observed. There are not enough details except 


```
Setting Chat History
--
Initializing FSI Agent
Creating FSI Agent - Start
Creating FSI Agent - Middle
Creating FSI Agent - End
Running FSI Agent with input = What kind of questions can FSI Agent answer?
[1m> Entering new AgentExecutor chain...[0m
Error raised by bedrock service: An error occurred (ValidationException) when calling the InvokeModel operation: The requested operation is not recognized by the service.
[ERROR] ValueError: Error raised by bedrock service: An error occurred (ValidationException) when calling the InvokeModel operation: The requested operation is not recognized by the service.Traceback (most recent call last):  File "/var/task/lambda_function.py", line 851, in handler    return dispatch(event)  File "/var/task/lambda_function.py", line 835, in dispatch    return genai_intent(intent_request)  File "/var/task/lambda_function.py", line 813, in genai_intent    output = invoke_fm(intent_request)  File "/var/task/lambda_function.py", line 796, in invoke_fm    raise e


```

**Comment by kyleblocksom** [comment]
Resolved with [Python 3.11, LangChain 1.28.64 patch](https://github.com/aws-samples/generative-ai-amazon-bedrock-langchain-agent-example/commit/1d39968eb3bafbc84f9f3f484f41353c2c593656).
